stopcovid/event_distributor/outbound_sms.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from typing import List
+from dataclasses import dataclass
+
+from stopcovid.dialog.types import DialogEvent
+from stopcovid.dialog.dialog import (
+    DrillCompleted,
+    AdvancedToNextPrompt,
+    FailedPrompt,
+    CompletedPrompt,
+    UserValidated,
+    DrillStarted,
+    UserValidationFailed,
+)
+from stopcovid.drills.localize import localize
+
+
+TRY_AGAIN = "{{incorrect_answer}}"
+
+USER_VALIDATION_FAILED_COPY = (
+    "Invalid Code. Check with your administrator and make sure you have the right code."
+)
+USER_VALIDATED_COPY = "You're all set! You should get your first drill soon."
+
+# We should template these for localization
+DRILL_COMPLETED_COPY = "Drill complete!"
+CORRECT_ANSWER_COPY = "Correct!"
+
+
+@dataclass
+class OutboundSMS:
+    event_id: str
+    phone_number: str
+    body: str
+
+
+def get_localized_messages(
+    dialog_event: DialogEvent, messages: List[str], **kwargs
+) -> List[OutboundSMS]:
+    language = dialog_event.user_profile.language
+    return [
+        OutboundSMS(
+            event_id=dialog_event.event_id,
+            phone_number=dialog_event.phone_number,
+            body=localize(message, language, **kwargs),
+        )
+        for message in messages
+    ]
+
+
+def get_messages_for_event(event: DialogEvent):  # noqa: C901
```

Comment:
Reminder: You need to add in delays between messages. You can do that in a subsequent PR.